changes/461.feature.rst

```diff
@@ -0,0 +1 @@
+Updated the exception message when calling Objective-C methods with invalid keyword arguments to include selector names.
```

Comment:
The error message has been improved when an Objective-C selector cannot be found matching the provided arguments.